src/dlfind/dlf_qts.f90

```diff
@@ -1716,7 +1716,7 @@ subroutine dlf_qts_get_hessian(trerun_energy)
       if(taskfarm_mode==1) call dlf_tasks_int_sum(status, 1)
       if(status > 0 ) then ! hessian was read by first task
         ! distribute it to others - only qts%ihessian_image
-        if(taskfarm_mode==1) call dlf_tasks_real_sum(qts%ihessian_image,neb%varperimage*neb%varperimage)
+        if(taskfarm_mode==1) call dlf_tasks_real_sum2(qts%ihessian_image,neb%varperimage*neb%varperimage)
```

Comment:
This will break the MPI parallel DL-FIND code. But since we don't use it that's fine.